### PR TITLE
[Integration Tests] Update a demo's config.json as part of integration test

### PR DESCRIPTION
**Pull request by dawoodkhan82**
# Description
When `pnpm test:browser` or `pnpm test:browser:full` is run from the `ui` the configs for each demo in the integration tests will be re-created.

Please include: 
* relevant motivation
* a summary of the change 
* which issue is fixed. 
* any additional dependencies that are required for this change.

Closes: # (issue)
#1937

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: demo/kitchen_sink/run.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def fn(
             (".", 0),
         ]
         + [(f"test", x / 10) for x in range(-10, 10)],  # HighlightedText
-        json.loads(JSONOBJ),  # JSON
+        json.loads(JSONOBJ),  # JSONåå
```
Comment (abidlabs) [inline demo/kitchen_sink/run.py:76]: Weird symbols showing up at the end for me?

**Comment by abidlabs** [comment]
@dawoodkhan82 this approach looks good, but the functional tests can't be run because `gradio` is not installed on the machine. Perhaps you have to do `pnpm exec pip install -r requirements.txt` first?

**Comment by abidlabs** [comment]
Actually probably better to add the steps to install `gradio` to this workflow directly: https://github.com/gradio-app/gradio/blob/21f9da9380b66420a598e79eb0b41feceba9077d/.github/workflows/ui.yml#L56

File: demo/kitchen_sink/run.py (hunk 1 above)
Comment (dawoodkhan82) [reply demo/kitchen_sink/run.py:76]: whoops good catch

File: .github/workflows/ui.yml [hunk 2]
```diff
@@ -53,6 +53,7 @@ jobs:
           node-version: 16
           cache: pnpm
           cache-dependency-path: ui/pnpm-lock.yaml
+      - run: pnpm exec pip install -r ../requirements.txt
```
Comment (abidlabs) [inline .github/workflows/ui.yml:56]: I think this should work, right?
```suggestion
      - run: pip install -r ../requirements.txt
```

**Comment by abidlabs** [comment]
Awesome @dawoodkhan82! Just one suggestion for simplification but otherwise LGTM

**Review by abidlabs — APPROVED** [review]

**Pull request merged**